tracee/tracee.go

```diff
@@ -723,6 +725,9 @@ func (t *Tracee) Run() error {
 	t.eventsPerfMap.Stop()
 	t.fileWrPerfMap.Stop()
 	t.printer.Epilogue(t.stats)
+
+	fmt.Printf("Written Files: %+v\n", t.writtenFiles)
```

Comment:
This is the correct location to save the written files,
but instead of printing them to stdout, can you please save them to a file in tracee's output dir (I think you did that in the previous commit, but here you only open and write to the file once)